index.ts

```diff
@@ -35,54 +61,99 @@ export class I2CPortMap extends Map<PortNumber, I2CPort> {
   }
 }
 
+/**
+ * I2CPort クラス定義
+ */
 export class I2CPort {
   private readonly _portNumber: PortNumber;
 
+  /**
+   * Creates an instance of GPIOPort.
+   * @param portNumber ポート番号
+   */
   constructor(portNumber: PortNumber) {
     this._portNumber = parseUint16(portNumber.toString());
   }
 
+  /**
+   * ポート番号取得処理
+   * @return 現在のポート番号
+   */
   get portNumber(): PortNumber {
     return this._portNumber;
   }
 
+  /**
+   * ポート名取得処理
+   * @return 現在のポート名
+   */
   get portName(): string {
     return `i2c-${this.portNumber}`;
   }
 
+  /**
+   * I2CSlave 接続デバイスオープン処理
+   * @param slaveAddress 接続デバイス情報のアドレス
+   * @return I2CSlaveDevice インスタンスの生成の完了
+   */
   async open(slaveAddress: I2CSlaveAddress): Promise<I2CSlaveDevice> {
     const bus = await openPromisified(this.portNumber).catch((error) => {
       throw new OperationError(error);
     });
 
     return {
       slaveAddress,
-      read8: (cmd) =>
-        bus.readByte(slaveAddress, cmd).catch((error) => {
+      /**
+       * @function
+       * I2c s/I2c/I2C 読み取り処理
```

Comment:
```suggestion
       * I2C 読み取り処理
```

で良いです。(私の説明が適切でなかったので申し訳ないです。`s/I2c/I2C` は `I2c` を `I2C` にしたほうがよいという意図で、`s/byte/bytes` は `byte` を `bytes` にしたほうがよいという意図でコメントしていました。)